src/NexusMods.App.UI/Pages/TextEdit/TextEditorPageViewModel.cs

```diff
@@ -138,7 +144,7 @@ public TextEditorPageViewModel(
                     };
 
                     Document = document;
-                    IsReadOnly = false;
+                    IsReadOnly = context.FileId.IsT1;
```

Comment:
This feels like a bit of a hack; but I'm not concerned about this for now, we can leave it as-is.

Eventually down the road, mod authors will want to be able to edit their own files; and they may try doing it in the App too, so there likely will be a case where you would want non-RO LibraryFiles. But such use case is very far from now, so I'm not concerned.